EMSBackendBridgeBundle/EventListener/RequestListener.php

```diff
@@ -48,36 +51,16 @@ public function onKernelException(GetResponseForExceptionEvent $event)
      */
     private function setEnvironment(Request $request)
     {
-        if (null === $this->requestEnvironment) {
+        if (null == $this->requestEnvironments) {
```

Comment:
why replace the identitary equal by a "simple" equal"?

Reply:
before it was just a string value, now requestEnvironments is an array. Maybe use a count if ?

Reply:
In that case I would suggest an !empty()